packages/grid/x-grid/src/tests/events.XGrid.test.tsx

```diff
@@ -0,0 +1,267 @@
+    });
+    it('should contains the cell isEditable', () => {
+      expect(eventArgs!.params.isEditable).to.equal(true);
+    });
+    it('should contains the element', () => {
+      expect(eventArgs!.params.element).to.equal(cell22);
+    });
+    it('should contains the cell model', () => {
+      expect(eventArgs!.params.row).to.equal(baselineProps.rows[1]);
+    });
+    it('should contains the cell index', () => {
+      expect(eventArgs!.params.rowIndex).to.equal(1);
+    });
+    it('should contains the column', () => {
+      expect(eventArgs!.params.colDef).to.equal(apiRef!.current.getColumnFromField('first'));
+    });
+    it('should contains the column index', () => {
+      expect(eventArgs!.params.colIndex).to.equal(1);
+    });
+    it('should contains the api', () => {
+      expect(eventArgs!.params.api).to.equal(apiRef.current);
+    });
+  });
+
+  describe('onCellClick', () => {
+    it('should bubble to the row', () => {
+      let events = '';
+      const addEventName = (name: string) => () => {
+        events += `${name}, `;
+      };
+      render(
+        <TestEvents
+          onCellClick={addEventName('cellClick')}
+          onRowClick={addEventName('rowClick')}
+        />,
+      );
+
+      const cell22 = getCell(1, 1);
+      fireEvent.click(cell22);
+      expect(events).to.equal('cellClick, rowClick, ');
+    });
+
+    it('should not bubble to the row if the column has disableEventBubbling', () => {
+      let events = '';
+      const addEventName = (name: string) => () => {
+        events += `${name}, `;
+      };
+      render(
+        <TestEvents
+          onCellClick={addEventName('cellClick')}
+          onRowClick={addEventName('rowClick')}
+          columns={baselineProps.columns.map((col) => ({
+            ...col,
+            disableClickEventBubbling: true,
+          }))}
+        />,
+      );
+
+      const cell22 = getCell(1, 1);
+      fireEvent.click(cell22);
+      expect(events).to.equal('cellClick, ');
+    });
+    it('should allow to stop propagation', () => {
+      let events = '';
+      const addEventName = (name: string) => () => {
+        events += `${name}, `;
+      };
+      const stopClick = (params, event) => {
+        event.stopPropagation();
+      };
+      render(<TestEvents onCellClick={stopClick} onRowSelected={addEventName('rowSelected')} />);
+
+      const cell22 = getCell(1, 1);
+      fireEvent.click(cell22);
+      expect(events).to.equal('');
+    });
+    it('should select a row by default', () => {
+      let events = '';
+      const addEventName = (name: string) => () => {
+        events += `${name}, `;
+      };
+      render(<TestEvents onRowSelected={addEventName('rowSelected')} />);
+
+      const cell22 = getCell(1, 1);
+      fireEvent.click(cell22);
+      expect(events).to.equal('rowSelected, ');
+    });
+    it('should not select a row if options.disableSelectionOnClick', () => {
```

Comment:
In the current convention, test cases have a blank line between each other. How about we add them here too?

```suggestion

    it('should not select a row if options.disableSelectionOnClick', () => {
```

x the other cases